contributors/design-proposals/apps/OBSOLETE_templates.md

```diff
@@ -1,3 +1,5 @@
+# OBSOLETE
```

Comment:
Should probably not be doing this here

Reply:
any suggestions for where/how to mark these deprecated?

Reply:
Let's wait for the metadata proposal.

https://docs.google.com/document/d/1ynmBMuDuT7yGzRscObB1KtgJj8ljYq0I5q4oshrJUCs/edit#